packages/xo-server-auth-ldap/src/index.js

```diff
@@ -183,8 +184,7 @@ export const testSchema = {
 // ===================================================================
 
 class AuthLdap {
-  constructor({ logger = noop, xo }) {
```

Comment:
What about the CLI?

Reply:
Right! Do you think it's necessary to be able to pass a custom logger?

Reply:
Functionally, I want logs to always be displayed when using the CLI.

Reply:
Using `@xen-orchestra/log` from the inside without passing a custom logger also displays the logs when using the CLI, so I can just remove the argument in `test-cli`.